src/locales/en/battle.json

```
@@ -32,6 +32,7 @@
   "attackMissed": "{{pokemonNameWithAffix}} avoided the attack!",
   "attackHitsCount": "Hit {{count}} time(s)!",
   "rewardGain": "You received\n{{modifierName}}!",
+  "rewardGainCount": "You received\n{{rewardCount}} {{modifierName}}!",
```

When wanting to use the `count` to use plurals in `modifierName` you have to use the [i18next nesting feature](https://www.i18next.com/translation-function/nesting#passing-options-to-nestings)

```suggestion
  "rewardGainCount": "You received\n{{count}} $t(modifierName, {\"count\": {{count}} })!",
```

----

Se my general review comment for more info about this

---

I changed rewardCount to count

---

the nested call is still necessary to make it future proof. soory :(